test/test_validation.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var assert = require("assert"),
```

Comment:
I understand your confusion. It's my fault and I apologise. I updated the main file, but didn't do the same with the test.